src/features/plans/components/PlaceItem.tsx

```
@@ -0,0 +1,62 @@
+  onSelect: () => void
+}
+
+export default function PlaceItem({ place, isRepresentative, onSelect }: PlaceItemProps) {
+  return (
+    <div
+      onClick={onSelect}
+      className={cn(
+        'flex gap-4 rounded-lg p-2 cursor-pointer transition-colors',
+        isRepresentative ? 'bg-primary/10 ring-1 ring-primary/30' : 'hover:bg-secondary/50',
+      )}
+    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)